### DISCUSSION TITLE
Per App settings

### DISCUSSION THREAD

**Discussion by daitarn** [Caldis/Mos#464] [Ideas]
Howdy,

as the title says, can a per app settings be implemented?

And a big Thank You for this handy little gem!

**Comment by harleymckenzie** [comment]
I can't remember when this was introduced but I feel like it's been possible to do this for a while by going to Preferences > Exceptions.

1. Click the + in the bottom left corner
2. Select an application (Running or select from Finder)
3. Click the ellipsis on the right-hand side of the application
4. Untick 'Inherit global settings' and configure it how you like
